miranda/hq/daily.py

```diff
@@ -0,0 +1,254 @@
+import csv
+import datetime as dt
+import json
+import re
+from pathlib import Path
+from typing import Optional
+from typing import Tuple
+from typing import Union
+
+import numpy as np
+import pandas as pd
+import pint
+import xarray as xr
+
+
+def units():
```

Comment:
Would it be possible to place this higher in a `utils` module so that the EC (and MELCC, eventually) scripts could make use of it? At the moment, I'm performing manual conversions of units for EC, but it would be good to expand this to other observation sources.

Reply:
Sure, there could be a context for each source (hq, eccc, etc).

Reply:
Ok. I'll open a ticket on that to look into later when I have time.